envsubst/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM chatwork/alpine-sdk:3.11
+
+ARG ENVSUBST_VERSION=1.1.0
+
+LABEL version="v${ENVSUBST_VERSION}"
+LABEL maintainer="sakamoto@chatwork.com"
+
+ADD https://github.com/a8m/envsubst/releases/download/v${ENVSUBST_VERSION}/envsubst-Linux-x86_64 /usr/local/bin/envsubst
+
+RUN chmod 755 /usr/local/bin/envsubst && \
+    apk update && \
```

Comment:
@cw-sakamoto 

Do you need `apk update`?

Reply:
🙈
I fixed in https://github.com/chatwork/dockerfiles/pull/241/commits/5ffde8a0b8235eee1ab82cc8b846d969267655b2
(`jq` is unneeded)